python-squid/src/test/java/org/sonar/python/tree/PythonTreeMakerTest.java

```diff
@@ -2205,6 +2220,27 @@ private void assertUnaryExpression(String operator, Tree.Kind kind) {
     // ensure every visit method of base tree visitor is called without errors
     BaseTreeVisitor visitor = new BaseTreeVisitor();
     tree.accept(visitor);
+    List<TokenType> ptt = Arrays.asList(PythonTokenType.NEWLINE, PythonTokenType.DEDENT, PythonTokenType.INDENT, GenericTokenType.EOF);
+    List<Token> tokenList = TreeUtils.tokens(tree);
+
+//    Token previous = null;
```

Comment:
I think you forgot to remove this commented code

Reply:
noted already